components/gitpod-protocol/src/public-api-converter.ts

```diff
@@ -375,4 +383,49 @@ export class PublicAPIConverter {
         result.defaultWorkspaceImage = settings.defaultWorkspaceImage || undefined;
         return result;
     }
+
+    toConfiguration(project: Project): Configuration {
+        const result = new Configuration();
+        result.id = project.id;
+        result.organizationId = project.teamId;
+        result.name = project.name;
+        result.cloneUrl = project.cloneUrl;
+        result.workspaceSettings = this.toWorkspaceSettings(project.settings?.workspaceClasses?.regular);
+        result.prebuildSettings = this.toPrebuildSettings(project.settings?.prebuilds);
+        return result;
+    }
+
+    toPrebuildSettings(prebuilds: import("./teams-projects-protocol").PrebuildSettings | undefined): PrebuildSettings {
```

Comment:
(nit) can go to import above

Reply:
It's only used once and the name would conflict